clippy_lints/src/pub_underscore_fields.rs

```diff
@@ -20,9 +20,15 @@ declare_clippy_lint! {
     /// ```
     /// Use instead:
     /// ```rust
-    /// struct FileHandle {
+    /// struct FileHandle_Foo {
```

Comment:
Mixing PascalCase with underscores hehe :cat2:
For future reference, you can use [attributes](https://doc.rust-lang.org/rustdoc/write-documentation/documentation-tests.html#attributes) like `should_panic` on your code blocks so that these name collisions don't matter.

Reply:
You can make it two separate code blocks
`````md
```rust
struct FileHandle { .. }
```
or
```rust
struct FileHandle { .. }
```
`````

Reply:
🤦‍♂️ yep totally missed this, updated it to be two code blocks!